lightningd/peer_control.c

```diff
@@ -622,6 +623,17 @@ static void gossipd_getpeers_complete(struct subd *gossip, const u8 *msg,
 			json_array_end(response);
 		}
 
+		
+		for (size_t i = 0; i < tal_count(ids); i++) {
```

Comment:
Should it not be up to `tal_count(nodes)`, since that is what is being iterated over?

Reply:
yes, will fix